library/src/main/scala/scala/scalajs/js/Object.scala

```diff
@@ -57,6 +57,38 @@ object Object extends Object {
   def apply(): Object = native
   def apply(value: Any): Object = native
 
+  /** <span class="badge badge-ecma6" style="float: right;">ECMAScript 6</span>
+   *
+   * The Object.assign() method is used to copy the values of all enumerable own
```

Comment:
Could you make sure to follow our coding styles for Scaladoc, explained at https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#scaladoc?

You'll also need to remove the blank line above this line for the initial sentence to correctly appear in the rendered Scaladoc.